### PR TITLE
Tanner Dale

### PR DESCRIPTION
**Pull request by TannerDale**

### CODE REVIEW

File: lib/event.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  def food_truck_names
+    @food_trucks.map do |truck|
+      truck.name
+    end
+  end
+
+  def food_trucks_that_sell(item)
+    @food_trucks.find_all do |truck|
+      truck.can_sell?(item)
```
Comment (mikedao) [inline lib/event.rb:22]: I like that you wrote a helper method, but we could have also used check_stock

File: lib/event.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      amount -= total_sold
+    end
+  end
+
+  def format_date(date)
+    day = date.day.to_s.rjust(2, "0")
+    month = date.month.to_s.rjust(2, "0")
+
+    "#{day}/#{month}/#{date.year}"
+  end
```
Comment (mikedao) [inline lib/event.rb:78]: Good. strftime would have simplified this

**Review by mikedao — COMMENTED** [review]
Great work!
Score: Pass

File: lib/event.rb (hunk 1 above)
Comment (TannerDale) [reply lib/event.rb:22]: Hmmm forgot about that one...

File: lib/event.rb (hunk 2 above)
Comment (TannerDale) [reply lib/event.rb:78]: Good to know! I'll look into that

**Pull request closed without merging**